src/Evolution/Initialization/Interface.hpp

```diff
@@ -41,67 +51,76 @@ namespace Initialization {
 template <typename System>
 struct Interface {
   static constexpr size_t dim = System::volume_dim;
-  using simple_tags =
-      db::AddSimpleTags<Tags::Interface<Tags::BoundaryDirectionsExterior<dim>,
-                                        typename System::variables_tag>>;
+  using simple_tags = db::AddSimpleTags<::Tags::Interface<
```

Comment:
same comments about moving these tags lists into `apply` or make private

Reply:
Leaving as is.